rules/scripts/bitstreams_workspace.py

```diff
@@ -401,10 +399,6 @@ def _ConstructBazelString(self, build_file: Path, key: str) -> str:
             manifest["schema_version"] = MANIFEST_SCHEMA_VERSION
 
         if manifest["schema_version"] == 2:
-            schema_path = os.path.join(MANIFESTS_DIR, "bitstreams_manifest.schema.json")
```

Comment:
Please leave this code in the script. If you need to make jsonschema optional for now, move the import here inside a try..catch, and gate the call behind a feature flag.

Reply:
Sure, will do. Out of interest, do you use this script from outside of Bazel?

Reply:
Yes, I use it outside bazel primarily for development and testing, but it can also be used to set up an offline cache.

Reply:
I've added this back behind a try/except but wasn't sure what you mean about a feature flag. Is this so that you'll get an error message if you ask for it to check the schema but don't have `jsonschema` installed?

Reply:
I just meant that you'd set some variable representing that schema validation is enabled, but since the only user is right here, maybe it's not necessary. I think this will do. Thanks! :)